File: services/reservation/src/main/java/com/example/iter/reservation/domain/repository/ReturnReceiptImageRepository.java
package com.example.iter.reservation.domain.repository;

import com.example.iter.reservation.domain.entity.ReturnReceiptImage;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ReturnReceiptImageRepository extends JpaRepository<ReturnReceiptImage, Long> {
    // 반납 증빙 이미지를 등록 순서대로 조회합니다.
    List<ReturnReceiptImage>
    findByReturnReceipt_IdOrderBySortOrderAscIdAsc(Long returnReceiptId);
}

File: services/reservation/src/main/kotlin/com/example/iter/reservation/domain/repository/ReturnReceiptImageRepository.kt
package com.example.iter.reservation.domain.repository

import com.example.iter.reservation.domain.entity.ReturnReceiptImage
import org.springframework.data.jpa.repository.JpaRepository

interface ReturnReceiptImageRepository : JpaRepository<ReturnReceiptImage, Long> {
    // 반납 증빙 이미지를 등록 순서대로 조회합니다.
    fun findByReturnReceipt_IdOrderBySortOrderAscIdAsc(returnReceiptId: Long): List<ReturnReceiptImage>
}
